platform/conf/conf.go

```diff
@@ -224,6 +224,40 @@ func (c *Conf) Bytes() []byte {
 	return []byte(c.String())
 }
 
+func (c *Conf) addSystemdUnitV1(name, contents string, enable bool) {
+	c.ignitionV1.Systemd.Units = append(c.ignitionV1.Systemd.Units, v1types.SystemdUnit{
+		Name: v1types.SystemdUnitName(name),
+		Contents: contents,
+		Enable: enable,
+	})
+}
+
+func (c *Conf) addSystemdUnitV2(name, contents string, enable bool) {
+	c.ignitionV2.Systemd.Units = append(c.ignitionV2.Systemd.Units, v2types.Unit{
+		Name: name,
+		Contents: contents,
+		Enable: enable,
```

Comment:
Some tests are still using 2.0 configs, should I update those to 2.1 in a seperate commit (or PR)? Without updating those changing this to Enabled causes ignition to fail with an unrecognized key.